templates/entry/_options.html.twig

```diff
@@ -1,136 +1,227 @@
-        <li>
-            <a href="{{ options_url('sortBy', 'active') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'active')}) }}">
-                {{ 'active'|trans }}
-            </a>
-        </li>
-        <li>
-            <a href="{{ options_url('sortBy', 'commented') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'commented')}) }}">
-                {{ 'commented'|trans }}
-            </a>
+    <menu class="options__filter">
+        <li class="dropdown">
+            <button aria-label="{{ 'sort_by'|trans }}"
+                    title="{{ 'sort_by'|trans }}"><i
+                        class="fa-solid fa-sort" aria-hidden="true"></i>
+                <span>{{ criteria.getOption('sort')|trans }}</span>
+            </button>
+            <ul class="dropdown__menu">
+                <li>
+                    <a href="{{ options_url('sortBy', 'top') }}"
+                    class="{{ html_classes({'active': criteria.getOption('sort') == 'top'}) }}">
+                        {{ 'top'|trans }}
+                    </a>
+                </li>
+                <li>
+                    <a href="{{ options_url('sortBy', 'hot') }}"
+                    class="{{ html_classes({'active': criteria.getOption('sort') == 'hot'}) }}">
+                        {{ 'hot'|trans }}
+                    </a>
+                </li>
+                <li>
+                    <a href="{{ options_url('sortBy', 'newest') }}"
+                    class="{{ html_classes({'active': criteria.getOption('sort') == 'newest'}) }}">
+                        {{ 'newest'|trans }}
+                    </a>
+                </li>
+                <li>
+                    <a href="{{ options_url('sortBy', 'active') }}"
+                    class="{{ html_classes({'active': criteria.getOption('sort') == 'active'}) }}">
+                        {{ 'active'|trans }}
+                    </a>
+                </li>
+                <li>
+                    <a href="{{ options_url('sortBy', 'commented') }}"
+                    class="{{ html_classes({'active': criteria.getOption('sort') == 'commented'}) }}">
+                        {{ 'commented'|trans }}
+                    </a>
+                </li>
+            </ul>
         </li>
-    </menu>
-    <menu class="options__view">
         <li class="dropdown">
             <button aria-label="{{ 'filter_by_time'|trans }}"
-                    title="{{ 'filter_by_time'|trans }}"><i
-                        class="fa-solid fa-clock" aria-hidden="true"></i> <span>{{ get_time_param_translated() }}</span>
+                    title="{{ 'filter_by_time'|trans }}">
+                <i aria-hidden="true" class="fa-solid fa-clock"></i>
+                {% if showFilterLabels == 'on' or (showFilterLabels == 'auto' and criteria.getOption('time') != '∞') %}
+                    <span>{{ criteria.getOption('time')|trans }}</span>
+                {% endif %}
```

Comment:
The default theme settings suggest, that the default is showing the labels. However if you do not accept `null`/`empty value` in this if, the labels will not be shown unless the theme setting is set to "on" by the user explicitely

Reply:
Got it. I fixed the variable set up at the top of the file to default to 'on' if we don't find a cookie.